source/fe/mapping_q_generic.cc

```diff
@@ -353,8 +353,9 @@ namespace internal
 
         // Construct the tensor product polynomials used as shape functions for the
         // Qp mapping of cells at the boundary.
+        QGaussLobatto<1> line_support_points(data.polynomial_degree+1);
         const TensorProductPolynomials<dim>
-        tensor_pols (Polynomials::generate_complete_Lagrange_basis(data.line_support_points.get_points()));
+        tensor_pols (Polynomials::generate_complete_Lagrange_basis(line_support_points.get_points()));
```

Comment:
It is called in the constructor of `InternalData` but now I realize it is also called in `do_transform_real_to_unit_cell` for degree 2 and higher. It won't show up in 3D for sure because setting up all the shape functions is definitely much more expensive but in 2D it's not so clear. I will turn it back in to be sure.